common/utils/mission.js

```diff
@@ -88,9 +88,15 @@ export function computeMissionStats(m, users) {
     u => u.id
   );
   const validatorIds = m.validations.map(v => v.submitterId);
+  const adminValidatorIds = m.validations
```

Comment:
```suggestion
  const adminValidatedForMemberIds = m.validations
```

ou quelque chose dans le style